src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -39,18 +39,7 @@ private string GetDebuggerDisplay()
         private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
-        protected NodeFlags flags
-        {
-            get => _nodeFlagsAndSlotCount.NodeFlags;
-            set => _nodeFlagsAndSlotCount.NodeFlags = value;
-        }
-
-        /// <inheritdoc cref="NodeFlagsAndSlotCount.SmallSlotCount"/>>
-        private byte _slotCount
-        {
-            get => _nodeFlagsAndSlotCount.SmallSlotCount;
-            set => _nodeFlagsAndSlotCount.SmallSlotCount = value;
-        }
+        internal NodeFlags Flags => _nodeFlagsAndSlotCount.NodeFlags;
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

It looks like we increased visibility of this API. Is this intentional?

Reply:
> See line 238. The property was always internal. 

Please place its definition to its original place then, in order to make the diff more meaningful

Reply:
And, please, restore the original style for the definition.

Reply:
Wfm